lab_1/main.py

```diff
@@ -87,27 +171,52 @@ def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> li
     right_n = 3
     --> [['man', 'is'], ['dog, 'cat']]
     """
-    pass
-
-
-def read_from_file(path_to_file: str) -> str:
+    adjacent_words = []
+    words = []
+    if type(tokens) == list and type(word) == str and type(left_n) == int and type(right_n) == int:
```

Comment:
Usually, there are check for input arguments and body function - on the same level.
No need for nesting.